server/middleware/setUpWebSecurity.ts

```diff
@@ -13,13 +13,7 @@ export default function setUpWebSecurity(): Router {
         directives: {
           defaultSrc: ["'self'"],
           // Hash allows inline script pulled in from https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/template.njk
-          scriptSrc: [
-            "'self'",
-            'code.jquery.com',
-            "'sha256-+6WnXIl4mbFTCARd8N3COQmT3bJJmo32N8q8ZSQAIcU='",
-            "'sha256-loNEcvOiwPZFEqTNJYE+/sdSv0X/6zDcCFyxAG+oGiw='",
```

Comment:
Firefox does not support unsafe-hashes for inline event handlers - so I've removed this and added some jquery in main (it supports inline script though - so I've left that bit in (from the gds template))